src/base/memory_monitor_sr.cpp

```diff
@@ -518,6 +532,14 @@ namespace cubperf
       }
   }
 
+  memory_monitor::memory_monitor (const char *server_name)
+    : m_server_name {server_name}, m_total_mem_usage {0}, m_aggregater {this}
+  {
+    /* TODO: this dummy modules will be changed when heap module is registered */
+    m_module[0] = std::make_unique<mmon_module> ("dummy", dummy_stat_info);
```

Comment:
I don't remember exactly, but there should've been some connections between definitions. 
```suggestion
    m_module[MMON_MODULE_DUMMY] = std::make_unique<mmon_module> (module_names[MMON_MODULE_DUMMY], dummy_stat_info);
```

Reply:
You're right. Done.